cmd/snaptel/task.go

```diff
@@ -234,6 +248,13 @@ func (t *task) setScheduleFromCliOptions(ctx *cli.Context) error {
 	if !ctx.IsSet("interval") && interval == "" && t.Schedule.Interval == "" {
 		return fmt.Errorf("Usage error (missing interval value); when constructing a new task schedule an interval must be provided")
 	}
+
+	countValStr := ctx.String("count")
+	if ctx.IsSet("count") || countValStr != "" {
+		if count, err := stringValToUint(countValStr); err == nil {
+			t.Schedule.Count = count
+		}
```

Comment:
Good point. I added returning the following error: `fmt.Errorf("Usage error (bad count format); %v", err)`, so it's similar to others error messages returning there.